starfish/experiment/experiment.py

```diff
@@ -265,39 +265,48 @@ def verify_version(cls, semantic_version_str: str) -> Version:
     def fov(
             self,
             filter_fn: Callable[[FieldOfView], bool]=lambda _: True,
+            key_fn: Callable[[FieldOfView], str]=lambda fov: fov.name,
```

Comment:
Is there a compelling reason for any alternative key?  I'm ok with making the fovs always sorted by name.

In the "distant" past, fovs were not guaranteed to have names, but as they are now, it might just make sense to always sort by name.

Reply:
> Is there a compelling reason for any alternative key? I'm ok with making the fovs always sorted by name.

The only other sorting that occurred to me was by physical coordinates in the case that the names are not helpful. `fov.name` was certainly easier in terms of a quick, stable sorting.

Reply:
Yeah, physical coordinates seemed like the only other plausible option, but that still seems unlikely since sorting a 3D space (R,C,Z) into 1D is just ... weird.

Reply:
@ttung: would you prefer the argument to be reverted?

Reply:
No, it does seem weird to me, but I'm okay leaving it in there. :)